kolibri/utils/options.py

```diff
@@ -64,7 +64,7 @@
     "Urls": {
         "CENTRAL_CONTENT_BASE_URL": {
             "type": "string",
-            "default": "http://studio.learningequality.org",
+            "default": "https://studio.learningequality.org",
```

Comment:
This was the other change that helped get the 23x content download speedup.

We recently started forcing HTTPS on Studio, and so every request was hitting a 301 redirect, which slowed things down a lot. Now that we have a good cert in there, it's safe to directly point at HTTPS.

Reply:
CC @aronasorman @indirectlylit

Reply:
I can't really recall why we didn't change it back to https after adding `pyOpenSSL`, but the original issue was: https://github.com/learningequality/kolibri/issues/2544. So I will double check on Ubuntu 14.04 to see if it works now (although I assume it's currently working).

Reply:
Quoting @jamalex: "the problem was the shared CF cert (used SNI, which Python 2.7.8 and below couldn't handle)", which was resolved by: "we paid CF to use our own certificate"

Reply:
@rtibbles thank you for the information! :)

Reply:
refs #2685